README.md

```diff
@@ -5,7 +5,7 @@
 Druid is a new Rust-native UI toolkit, still in early stages. Its main
 goal is performance, also aiming for small binary size and compile time,
 fast startup, and very easy build configuration (just `cargo run`). It
```

Comment:
I would replace this sentence with, 'It current supports Windows and macOS, with GNU/Linux support planned.'

Reply:
fixed